src/networking.c

```diff
@@ -3219,6 +3247,23 @@ NULL
         /* If this client has to be closed, flag it as CLOSE_AFTER_REPLY
          * only after we queued the reply to its output buffers. */
         if (close_this_client) c->flags |= CLIENT_CLOSE_AFTER_REPLY;
+    } else if (!strcasecmp(c->argv[1]->ptr, "count")) {
+        sds o = NULL;
+        if (c->argc == 4 && !strcasecmp(c->argv[2]->ptr, "user")) {
+            user = ACLGetUserByName(c->argv[3]->ptr,
+                                    sdslen(c->argv[3]->ptr));
+            if (user == NULL) {
+                addReplyErrorFormat(c, "No such user '%s'",
+                                    (char *) c->argv[3]->ptr);
+                return;
+            }
+            o = getUserClientsInfoString(user);
```

Comment:
i don't understand. this is a CLIENT **COUNT** command, that returns a verbatim string with a CLIENT INFO style string on all clients that use a certain USER?

i.e. that's a variant of a filtered CLIENT LIST, not a CLIENT COUNT.

anyway, we can argue about the keeping the list of clients per user, to optimize certain loops, but as far as the interface here it seems wrong, and in any case, we wanna take a more holistic approach to all the client related commands, like we described in https://github.com/redis/redis/issues/12311.